frontend/src/LandingPage.jsx

```diff
@@ -127,61 +131,61 @@ export default function LandingPage() {
   return (
     <div className={classes.landingPage}>
       <nav className={classes.nav}>
-        <Link to="/" className={classes.siteTitle}>
-        </Link>
+        <Link to='/' className={classes.siteTitle}></Link>
         <ul className={classes.navList}>
           <li className={classes.navListItem}>
-            <Box component="span" onClick={handleAccordionToggle}></Box>
+            <Box component='span' onClick={handleAccordionToggle}></Box>
 
-            <Box component="span">
+            <Box component='span'>
               <Button
                 className={classes.aboutLink}
                 onClick={() => setOpen(true)}
                 style={{ color: '#fff' }}
-              >
-              </Button>
+              ></Button>
             </Box>
           </li>
         </ul>
       </nav>
 
-      <div className={classes.mainContent}>
-        <Link to="/home" className={classes.buttonContainer} style={{ textDecoration: 'none' }}>
-          <Button
-            variant="contained"
-            sx={{
-              color: 'white',
-              padding: '15px 30px',
-              borderRadius: '10px',
-            }}
-          >
-            Create Simulation
-          </Button>
-        </Link>
+      {isLoading ? (
```

Comment:
Only change here is that isLoading ? , was added in order to add loading functionality.